runtime/cudaq/platform/default/rest_server/helpers/RestRemoteClient.cpp

```diff
@@ -279,6 +279,12 @@ class RemoteRestRuntimeClient : public cudaq::RemoteRuntimeClient {
 /// REST client submitting jobs to NVCF-hosted `cudaq-qpud` service.
 class NvcfRuntimeClient : public RemoteRestRuntimeClient {
 private:
+  // None: Don't log; Info: basic info; Trace: Timing data per invocation.
+  enum LogLevel : int { None = 0, Info, Trace };
```

Comment:
Nit: I would recommend use of scoped enumerations.
```suggestion
  enum class LogLevel : int { None = 0, Info, Trace };
```